src/test/java/it/gov/pagopa/payment/controller/payment/BarCodePaymentControllerIntegrationTest.java

```diff
@@ -144,7 +154,8 @@ void authorizeTransaction_TooManyRequestThrownByRewardCalculator() throws Except
         TransactionBarCodeResponse trxCreated = createTrxSuccess(trxRequest, USERID);
 
         // Authorizing transaction but obtaining Too Many requests by reward-calculator
-        extractResponse(authTrx(trxCreated.getTrxCode(), authPaymentDTO, MERCHANTID), HttpStatus.TOO_MANY_REQUESTS, null);
+        Object response = extractResponse(authTrx(trxCreated.getTrxCode(), authPaymentDTO, MERCHANTID), HttpStatus.TOO_MANY_REQUESTS, null);
+        assertNull(response);
```

Comment:
this assert null is useless! the extractResponse test method will always return null if "null" is the expected configured class! thus you are testint the extractResponse method this way! we are not interested on it!
If you want to check if there is not body, you should instead modify the extractResponse itself to check the empty body!